lib/utils.js

```diff
@@ -1,38 +1,131 @@
 const camelcase = require('camelcase');
+const colorConvert = require('color-convert');
 const colorNamer = require('color-namer');
+const fs = require('fs').promises;
+
+const isArtboardColor = color => color._class === 'artboardColor';
+exports.isArtboardColor = isArtboardColor;
+exports.isDocumentColor = color =>
+  color._class === 'color' || color._class === 'MSImmutableColorAsset';
+
+const fsErrorHandler = err => {
```

Comment:
If you don't export this and reuse it, write it inline. Don't abstract for the reason to abstract.